.eslintrc.json

```
@@ -1,47 +1,52 @@
 {
-  "root": true,
-  "ignorePatterns": [
-    "projects/**/*"
-  ],
-  "overrides": [
-    {
-      "files": [
-        "*.ts"
-      ],
-      "extends": [
-        "eslint:recommended",
-        "plugin:@typescript-eslint/recommended",
-        "plugin:@angular-eslint/recommended",
-        "plugin:@angular-eslint/template/process-inline-templates"
-      ],
-      "rules": {
-        "@angular-eslint/directive-selector": [
-          "error",
-          {
-            "type": "attribute",
-            "prefix": "app",
-            "style": "camelCase"
-          }
-        ],
-        "@angular-eslint/component-selector": [
-          "error",
-          {
-            "type": "element",
-            "prefix": "app",
-            "style": "kebab-case"
-          }
-        ]
-      }
-    },
-    {
-      "files": [
-        "*.html"
-      ],
-      "extends": [
-        "plugin:@angular-eslint/template/recommended",
-        "plugin:@angular-eslint/template/accessibility"
-      ],
-      "rules": {}
-    }
-  ]
+    "root": true,
+    "ignorePatterns": ["projects/**/*"],
+    "overrides": [
+        {
+            "extends": ["eslint:recommended", "google"],
+            "rules": {}
+        },
+        {
+            "extends": ["prettier"],
+            "plugins": ["prettier"],
+            "rules": {
+                "prettier/prettier": ["error"]
+            }
+        },
```

Тут вот какой нюанс имеется:
`overrides` предназначен для добавления определенных правил для файлов подходящих по маскам указанных в `files` (`"files": ["*.ts"]`). Соответсвенно конфиги лежащие в `overrides`, но не имеющие `files` - должны быть вынесены в общие правила:
```suggestion
    "extends": ["eslint:recommended", "google", "prettier"],
    "plugins": ["prettier"],
    "rules": {
        "prettier/prettier": ["error"]
    }
    "overrides": [
```

---

понял, вот еще вопрос: общие правила это на одном уровне с overrides?

---

Yep